message_definitions/v1.0/common.xml

```diff
@@ -3352,6 +3352,7 @@
       <field type="uint32_t" name="v_acc" units="mm">Altitude uncertainty. Positive for up.</field>
       <field type="uint32_t" name="vel_acc" units="mm">Speed uncertainty. Positive for up.</field>
       <field type="uint32_t" name="hdg_acc" units="degE5">Heading / track uncertainty</field>
+      <field type="int32_t" name="yaw" units="cdeg">Yaw in earth frame from north. Use 0 if this GPS does not provide yaw. Use -1 if this GPS is configured to provide yaw and is currently unable to provide it. Use 36000 for north</field>
```

Comment:
If this is going to be a 4 byte value it should be a float with no scaling, if we are going to do a max value of 36000 then this should be `uint16_t` to save space.

Reply:
we need the zero and -1 values separately so we can distinguish the cases of "no yaw supported" and "yaw not currently available".
If uint16_t we could do:
 0: yaw not supported
 65535: no yaw available
 1 to 36000: yaw value
I prefer the uint32_t, but that would also work

Reply:
changed to uint16_t as above